ic-helpers/src/tokens.rs

```diff
@@ -264,7 +264,7 @@ impl Tokens256 {
 
     /// Rounded square root of the value.
     pub fn sqrt(&self) -> Self {
-        Self(self.0.sqrt_vartime())
+        Self(self.0.sqrt())
```

Comment:
This should be reverted. If you have a compilation error remove the Cargo.lock

Reply:
Done